src/rpc/blockchain.cpp

```diff
@@ -1283,6 +1283,10 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             "  \"chainwork\": \"xxxx\"     (string) total amount of work in active chain, in hexadecimal\n"
             "  \"pruned\": xx,             (boolean) if the blocks are subject to pruning\n"
             "  \"pruneheight\": xxxxxx,    (numeric) lowest-height complete block stored\n"
+            "  \"initialblockdownload\":   (boolean) Is in initial block download\n"
+            "  \"datareceived\": xxxxxx,   (numeric) Data recieved in bytes\n"
+            "  \"datasent\": xxxxxx,       (numeric) Data sent in bytes\n"
```

Comment:
These two (`datareceived` and `datasent`) do not belong to blockchain state at all imo, you can get this data via `getnettotals` rpc.

Reply:
Yup realized that. haha will remove.